.npmrc

```diff
@@ -0,0 +1,3 @@
+@razorpay:registry=https://npm.pkg.github.com/
+//npm.pkg.github.com/:always-auth=true
+//npm.pkg.github.com/:_authToken=${GITHUB_ACCESS_TOKEN}
```

Comment:
can you try adding this inside `packages/blade-coverage-plugin/.npmrc` once and see if it works? 

on CI we auto-generate `.npmrc` on root here based on which registry we're publishing on so not sure if this might mess up with that.

Reply:
If I add .npmrc inside package then install fails.

Reply:
ahh makes sense. this should work then

Reply:
Would this be an issue for non-razorpay developers who are trying to run blade locally now?

Reply:
Yes it might be but based on my discussion with @saurabhdaware He told me that every developer has github_access_token, it is just that it will act as one more friction point. 

This PR was created @chaitanyadeorukhkar for @saurabhdaware to debug how to solve this.

Reply:
Will schedule office hours next week to discuss how to solve this, it might actually be solved after the fix you had raised @chaitanyadeorukhkar